pkg/kubelet/cpumanager/policy.go

```diff
@@ -24,6 +24,7 @@ import (
 // Policy interface used by cpu manager, shall implement logic for cpu to pod
 // assignment
 type Policy interface {
+	IsUnderPressure() bool
```

Comment:
Not opinionated enough to postpone merging this, but IMO it would match the existing responsibilities. The idea would be that the policy can set the pressure bit in the state, so the Manager can just consult the state about CPU pressure instead of asking the Policy (similar to how the reconcile loop works already).